packages/core/src/views/account-center/Minimized.svelte

```diff
@@ -40,15 +40,18 @@
       ? firstAccount.balance[firstAddressAsset]
       : null
 
-  $: primaryChain = primaryWallet.chains[0]
+  $: primaryChain =
+    primaryWallet && primaryWallet.chains && primaryWallet.chains.length
+      ? primaryWallet.chains[0]
+      : undefined
 
   $: validAppChain = chains.find(({ id, namespace }) =>
     primaryChain
       ? id === primaryChain.id && namespace === primaryChain.namespace
       : false
   )
 
-  $: defaultChainStyles = getDefaultChainStyles(primaryChain.id)
+  $: defaultChainStyles = getDefaultChainStyles((primaryChain && primaryChain.id) ? primaryChain.id : undefined)
```

Comment:
```suggestion
  $: defaultChainStyles = getDefaultChainStyles(primaryChain && primaryChain.id)
```
Could this be shortened to the above since `primaryChain` would be `undefined` if it does not have the `id` property?